web/pandas/pdeps/0005-no-default-index-mode.md

```diff
@@ -0,0 +1,351 @@
+# PDEP-5: NoRowIndex
+
+- Created: 14 November 2022
+- Status: Draft
+- Discussion: [#49693](https://github.com/pandas-dev/pandas/pull/49693)
+- Author: [Marco Gorelli](https://github.com/MarcoGorelli)
+- Revision: 2
+
+## Abstract
+
+The suggestion is to add a ``NoRowIndex`` class. Internally, it would act a bit like
+a ``RangeIndex``, but some methods would be stricter. This would be one
+step towards enabling users who do not want to think about indices to not need to.
```

Comment:
I find the above sentence awkward.  Maybe "This would be one step towards enabling users who do not want to think about indices".